.github/workflows/build-and-push-assets.yml

```
@@ -137,7 +147,21 @@ jobs:
       TAG_BRANCH_NAME: ''                              # we'll override if the push is for tag
       LOCK_FILE: ''                                    # we'll override after checking files
       NO_CHANGES: ''                                   # we'll override if no changes to commit
+      BUILT_BRANCH_SUFFIX: '-built'
+      RELEASE_BRANCH_NAME: 'release'
+      USE_BUILT_BRANCH: ${{ inputs.USE_BUILT_BRANCH }}
+      USE_RELEASE_BRANCH: ${{ inputs.USE_RELEASE_BRANCH }}
     steps:
+      - name: BUILT_BRANCH_SUFFIX deprecation notice
+        if: ${{ inputs.BUILT_BRANCH_SUFFIX != '' }}
+        run: |
+          echo "::notice::The BUILT_BRANCH_SUFFIX variable is deprecated and will be removed soon. If you use it, please update your workflow to use USE_BUILT_BRANCH."
+          echo "USE_BUILT_BRANCH=true" >> $GITHUB_ENV
+      - name: RELEASE_BRANCH_NAME deprecation notice
+        if: ${{ inputs.RELEASE_BRANCH_NAME != '' }}
+        run: |
+          echo "::notice::The RELEASE_BRANCH_NAME variable is deprecated and will be removed soon. If you use it, please update your workflow to use USE_RELEASE_BRANCH."
+          echo "USE_RELEASE_BRANCH=true" >> $GITHUB_ENV
```

Same here:
```suggestion
      - name: Deprecation warning
        if: ${{ inputs.RELEASE_BRANCH_NAME }}
        run: |
          echo ":: warning::The RELEASE_BRANCH_NAME input is deprecated and will be removed soon. If you use it, please update your workflow to use USE_RELEASE_BRANCH."
          echo "USE_RELEASE_BRANCH=true" >> $GITHUB_ENV

```
(Does GitHub Actions support steps with identical names ("Deprecation warning")?)